workers/ohsome_quality_analyst/oqt.py

```diff
@@ -135,15 +147,18 @@ async def create_report(
         Report
     """
     report_class = name_to_class(class_type="report", name=report_name)
-    report = report_class(bpolys=bpolys, dataset=dataset, feature_id=feature_id)
+    report = report_class(
+        bpolys=bpolys, dataset=dataset, feature_id=feature_id, fid_field=fid_field
+    )
     report.set_indicator_layer()
     for indicator_name, layer_name in report.indicator_layer:
         indicator = await create_indicator(
             indicator_name,
             layer_name,
```

Comment:
Why didn't you change these two to named parameters?

Reply:
Because they are required by the function `create_indicator` regardless of the other parameters, which have default values.